src/app/app.routing.ts

```diff
@@ -1,12 +1,16 @@
-import { NgModule } from '@angular/core';
-import { RouterModule, Routes } from '@angular/router';
+import {NgModule} from '@angular/core';
+import {RouterModule, Routes} from '@angular/router';
```

Comment:
are you using prettier for formating?
![image](https://user-images.githubusercontent.com/55179251/225417141-bef6158b-5f42-4807-a493-15647129575a.png)